charts/m4d/values.yaml

```diff
@@ -188,4 +188,57 @@ clusterScoped: true
 
 # Set to true to enable socat in the manager pod to forward
 # traffic from a localhost registry. Used only for development.
-integrationTests: false
\ No newline at end of file
+integrationTests: false
+
+# Values set in m4d-config ConfigMap.
+config:
+  # Vault related values set in m4d-config ConfigMap.
+  vault:
+    # Set to the Vault address.
+    address: "http://vault.m4d-system:8200/"
```

Comment:
side note - we will need a way to remove m4d-system from the default value at some point.